static/scripts/onboarding/onboarding.ts

```diff
@@ -552,12 +471,9 @@ const step2Handler = async () => {
 };
 
 const init = async () => {
-  let conf = await getConf(true);
-  if (conf !== undefined) {
+  if (defaultConf !== undefined) {
     try {
-      conf = JSON.parse(conf);
-      defaultConf = conf as IConf;
-      defaultConf["private-key-encrypted"] = "";
+      defaultConf[PRIVATE_ENCRYPTED_KEY_NAME] = "";
```

Comment:
I am pretty sure that we changed our strategy to be either `null` or the correct string for these situations.

Reply:
Changed it to `undefined` (type is `string | undefined` with the current conf from Ubiqubot repo)

Reply:
I prefer `null` as it is unambiguously empty vs `undefined` which could be a mistake. Why did you choose `undefined`?

Reply:
I didn't, it comes from the types in `ubiquibot-config-default.ts`